docs/source/release/v3.9.0/ui.rst

```diff
@@ -13,6 +13,7 @@ Instrument View
 ###############
 
 - New peak comparison tool on the pick tab. The user can select two peaks and information relating to their properties and the angles between them.
+- New tool for computing in-plane and out-of-plane angles in Qlab space for crystal alignment.
```

Comment:
Where is the documentation of what the angles mean? Do you have some formulas to compare to? My guess is that 4 peaks is too much. I would use 2 peaks + sample to define plane, but I am open to be persuaded (that's why I need documentation) that this is the correct approach